### PR TITLE
Increase order price font size and colors for better contrast

### LINKED ISSUE #17551: The typo of the value of an order considered as paid is still small.
The green style of the value when an order is considered as paid is still small.
See in the screenshot below.

download (3)

_Originally posted by @MatShir in https://github.com/PrestaShop/PrestaShop/issues/15971#issuecomment-582508738_

### PR DESCRIPTION
**Pull request by atomiix**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | The font size of the price for a paid order was too small. The font color for some default order status was not white as expected. This PR fixes it.
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #17551
| How to test?  | In the migrated Order list pages, the price for a paid order should have the same font size as a price for a non-paid order. Every migrated status colors should have white text color

## Order States color update

While fixing the white text color on default status, we noticed that 2 status color were not the same as in the original design. This PR fixes it too.

Here are the color changes : 
- #34219E -> #34209E (Awaiting payment)
- #2ECC71 -> #01B887 (Accepted payment)

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17568)

### CODE REVIEW

**Review by Progi1984 — APPROVED** [review]

File: admin-dev/themes/default/template/helpers/list/list_content.tpl [hunk 1]
```diff
@@ -64,7 +64,7 @@
 				{if isset($params.badge_warning) && $params.badge_warning && isset($tr.badge_warning) && $tr.badge_warning == $params.badge_warning}<span class="badge badge-warning">{/if}
 				{if isset($params.badge_danger) && $params.badge_danger && isset($tr.badge_danger) && $tr.badge_danger == $params.badge_danger}<span class="badge badge-danger">{/if}
 				{if isset($params.color) && isset($tr[$params.color])}
-					<span class="label color_field" style="background-color:{$tr[$params.color]};color:{if Tools::getBrightness($tr[$params.color]) < 128}white{else}#383838{/if}">
+					<span class="label color_field" style="background-color:{$tr[$params.color]};color:{if Tools::getBrightness($tr[$params.color]) < 130}white{else}#383838{/if}">
```
Comment (Progi1984) [inline admin-dev/themes/default/template/helpers/list/list_content.tpl]:
```suggestion
					<span class="label color_field" style="background-color:{$tr[$params.color]};color:{if Tools::getBrightness($tr[$params.color]) < constant('PrestaShop\PrestaShop\Core\Util\ColorBrightnessCalculator::BRIGHT_COLOR_MIN')}white{else}#383838{/if}">
```

File: src/Core/Util/ColorBrightnessCalculator.php [hunk 2]
```diff
@@ -34,7 +34,7 @@ final class ColorBrightnessCalculator
     /**
      * Minimum color value after which it's considered bright
      */
-    const BRIGHT_COLOR_MIN = 129;
+    const BRIGHT_COLOR_MIN = 130;
```
Comment (Progi1984) [inline src/Core/Util/ColorBrightnessCalculator.php]:
```suggestion
    public const BRIGHT_COLOR_MIN = 130;
```

**Review by Progi1984 — CHANGES_REQUESTED** [review]
Please some changes

File: classes/Tools.php [hunk 3]
```diff
@@ -2344,6 +2345,11 @@ public static function getBrightness($hex)
         return (($r * 299) + ($g * 587) + ($b * 114)) / 1000;
     }
 
+    public static function isBright($hex)
+    {
+        return (new ColorBrightnessCalculator())->isBright($hex);
```
Comment (PierreRambaud) [inline classes/Tools.php]: Could you add a way to don't reinstanciate `ColorBrightnessCalculator` each time this function is called please :)

File: src/Core/Util/ColorBrightnessCalculator.php [hunk 4]
```diff
@@ -34,7 +34,7 @@ final class ColorBrightnessCalculator
     /**
      * Minimum color value after which it's considered bright
      */
-    const BRIGHT_COLOR_MIN = 129;
+    private const BRIGHT_COLOR_MIN = 130;
```
Comment (PierreRambaud) [inline src/Core/Util/ColorBrightnessCalculator.php]: Don't make it private. It's a BC :thinking:

**Review by Progi1984 — DISMISSED** [review]

**Review by NeOMakinG — DISMISSED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by SD1982** [comment]
This Pr correctly increase the order price font size : 

**Before :** 
Order price font size #17568 before PR

**After :** 
Order price font size #17568 after PR

Thanks @atomiix

**Comment by NeOMakinG** [comment]
And what about the white color ? @SD1982

**Comment by atomiix** [comment]
@NeOMakinG looks like labels have white text on the "After" screenshot 😄

**Comment by NeOMakinG** [comment]
Yes but there one @atomiix  

```
#34219E -> #34209E (Awaiting payment)
#2ECC71 -> #01B887 (Accepted payment)
```

But well we can merge but the QA seems not fully done as he looked only for the font-size change 🤓

**Comment by SD1982** [comment]
@NeOMakinG  , @atomiix 

> And what about the white color ? @SD1982

The white color for status label font size was a part of this PR : 

Order price font size #17568 PR description

And the color change of the labels has also been verified 

But thank you anyway for notifying me I therefore take the opportunity to make a double check on this PR !

**Pull request merged**